soh/soh/Enhancements/randomizer/dungeon.h

```diff
@@ -76,8 +79,8 @@ typedef enum {
     SHADOW_TEMPLE,
     BOTTOM_OF_THE_WELL,
     ICE_CAVERN,
-    GERUDO_TRAINING_GROUNDS,
-    GANONS_CASTLE
+    GANONS_CASTLE,
+    GERUDO_TRAINING_GROUNDS
```

Comment:
Why was this changed?

Reply:
When making the check in item_pool.cpp (`for (uint8_t i = SCENE_DEKU_TREE; i <= SCENE_GERUDO_TRAINING_GROUND; i++) {`) I noticed they were in a different order than the vanilla scene nums, so I rectified it.